docs/experiment/guides/debugging-sample-ratio-mismatch.md

```diff
@@ -0,0 +1,73 @@
+<p></p>
+#### 4. How much variant jumping (same user seeing both treatment and control) was there? Is it more than 5%? 
+<p style="display:inline;">You can find this graph in the Monitor tab of an experiment. The issue with variant jumping is that you do not know which variant to attribute the metric to. If a users sees control, then treatment, and then does the conversion event, did the user do the conversion event because of the treatment or did the user just need time to think about if they wanted to buy the item and the treatment had no effect on their probability of purchase? If there is variant jumping, is it because of anonymous users (people constantly logging in and out) or changing device ids? You can see this by looking at the <a href="https://help.amplitude.com/hc/en-us/articles/229313067-Look-up-event-data-for-individual-users">User Stream</a>.</p>
+<p></p>
+#### 5. Do you see a big difference in the conversion percentage between variants for the Assignment event -> Exposure event funnel?
+<p style="display:inline;">You can find this graph in the Monitor tab of an experiment. Plug in the conversion rates and sample size in <a href="https://www.socscistatistics.com/tests/ztest/default2.aspx">here</a> and see if its statistically significant or not at the 95% significance level. The conversion rates between variants should be similar (within randomness) because the assignment event should randomly split users into two equal cohorts. Since exposure event gets sent right before the user experiences a variant, there should be no difference between users since they have the same user experience from the time the assignment event is sent ot the time the exposure event is sent.</p>
+<p></p>
+#### 6. Did you add/remove a variant in the middle of an experiment?
+<p style="display:inline;">This is just a special case of #2.</p>
+<p></p>
+#### 7. Was sticky bucketing turned on and off in the middle of the experiment?
+<p style="display:inline;">Generally, once you start an experiment you should not change anything.</p>
+<p></p>
+#### 8. Are there a lot of users in the ["Individual Users Allocation"](../general/evaluation/implementation.md#Individual-inclusions) section?
```

Comment:
"8. Did you [individually allocate](../../general/evaluation/implementation.md#Individual-inclusions) a large number of users?"